apps/i18n/common/en_us.json

```diff
@@ -19,6 +19,7 @@
   "clearPuzzleConfirmHeader": "Are you sure you want to start over?",
   "codeMode": "Code",
   "codeTooltip": "See generated JavaScript code.",
+  "completedWithoutRecommendedBlock": "Congratulations! You completed Puzzle {puzzleNumber}. (But you could use a different block for stronger code.)",
```

Comment:
Hmmm...stronger just seems like a more awkward way of saying better. My vote would probably be to just say better code since that's what we mean. If we really don't like that, maybe avoid all judgement and just say "But you could have used a different block."?